app/jobs/export_cocina_json_job.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+##
+# A job that exports line-oriented JSON for one or more objects
+# @param [Integer] bulk_action_id GlobalID for a BulkAction object
+# @param [Hash] params additional parameters that an Argo job may need
+class ExportCocinaJsonJob < GenericJob
+  def perform(bulk_action_id, params)
+    super
+
+    File.open(json_download_path, "w") do |jsonl_file|
+      with_items(params[:druids], name: "Export cocina JSON") do |cocina_object, success, failure|
```

Comment:
I can't remember how the `name` param is used, so maybe it's not worth doing here, but `cocina` is capitalized most other strings.

Reply:
It shows up in the bulk action's log, mainly for failures, e.g. `2023-08-30 17:33:17 UTC Export cocina JSON failed Dor::Services::Client::NotFoundResponse ...`

Agree it's more consistent to capitalize, so doing that, thanks!